third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1100,7 +1151,9 @@ static void SetFunctionProperty(v8::Isolate* isolate, v8::Local<v8::Object> obj,
     function_template->GetFunction(context).ToLocalChecked();
 
   v8::Local<v8::String> name_string = ToV8String(isolate, name);
-  obj->Set(context, name_string, function).Check();
+  obj->DefineOwnProperty(isolate->GetCurrentContext(), name_string, function,
```

Comment:
If you have `DefineProperty` above return a `Local<String>` then this code can be replaced with a call to that.

Reply:
Hmm, I'll rework this to remove some of the duplicate cruft but returning a Local<String> seems kind of weird and I'd like to avoid that.